vulnerabilities/helpers.py

```diff
@@ -100,9 +118,41 @@ def requests_with_5xx_retry(max_retries=5, backoff_factor=0.5):
     return session
 
 
-def contains_alpha(string):
-    """
-    Return True if the input 'string' contains any alphabet
-    """
+def nearest_patched_package(vulnerable_packages, resolved_packages):
```

Comment:
This is where the magic happens, again it has poor naming. Help.

Reply:
we will fix that in a separate ticket.